app/controllers/application.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+export default Ember.Controller.extend({
+    //Inject the service throughout the project
+    session: Ember.inject.service('session'),
+
+
+
+    //Call method to invalidate session and log the user out
+    actions: {
+    invalidateSession() {
```

Comment:
I wonder if there's a way to add this action to a `navbar` controller 🤔 if not then this place is as good as any!